Dockerfile

```diff
@@ -22,5 +24,8 @@ COPY dockerbuild/vhost.conf /etc/apache2/sites-enabled/
 # ErrorLog inside a VirtualHost block is ineffective for unknown reasons
 RUN sed -i -E 's@ErrorLog .*@ErrorLog /proc/self/fd/2@i' /etc/apache2/apache2.conf
 
+ADD https://github.com/silinternational/config-shim/releases/latest/download/config-shim_0.0.2-pre.3_linux_amd64.tar.gz config-shim.tar.gz
```

Comment:
I see this URL uses both "latest" and "0.0.2-pre". As new versions are released, do you know if that will cause a problem with this URL?

Reply:
Good question. According to [this](https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases) I don't believe it will be a problem. I may also change the release script to remove the version number from the `tar.gz` file just to make it simpler.

Reply:
Okay, thanks. Just wanted to ask.